Examples/Python/src/Json.cpp

```diff
@@ -148,5 +149,19 @@ void addJson(Context& ctx) {
           return pDetector;
         }));
   }
+
+  {
+    auto sjOptions = py::class_<ActsExamples::JsonSurfacesReader::Options>(
+                         mex, "SurfaceJsonOptions")
+                         .def(py::init<>());
+
+    ACTS_PYTHON_STRUCT_BEGIN(sjOptions,
+                             ActsExamples::JsonSurfacesReader::Options);
+    ACTS_PYTHON_MEMBER(inputFile);
+    ACTS_PYTHON_MEMBER(jsonEntryPath);
+    ACTS_PYTHON_STRUCT_END();
+
+    mex.def("readSurfaceFromJson", ActsExamples::JsonSurfacesReader::read);
+  }
```

Comment:
Can't this be exposed as the actual type, with a nested `Options` class and the `read` method?

Reply:
`JsonSurfacesReader` is a namespace - how would I do that then?

Reply:
Hm ok maybe overly complicated, but it could be modeled with a dummy class I guess. Not important.